.github/workflows/pull-db-tests.yml

```diff
@@ -251,3 +263,16 @@ jobs:
         env:
           TAGS: bindata
           USE_REPO_TEST_DIR: 1
+
+  test-coverage-check:
+    if: needs.files-changed.outputs.backend == 'true' || needs.files-changed.outputs.frontend == 'true'
```

Comment:
I still think the frontend dependency is not right in these backend jobs, see https://github.com/go-gitea/gitea/pull/25150 because frontend (js,css) can not influence backend in any way.

Reply:
They will affect web integration tests because some of these tests will parse output HTML

Reply:
But `frontend` does not produce/alter any HTML responses.

Reply:
But templates and vue will? They will change the dom nodes dynamically?

Reply:
Yes, but these modifications are not done when the backend tests run as backend tests do not run any javascript.

Reply:
Yes, [e2e tests](https://github.com/go-gitea/gitea/blob/main/.github/workflows/pull-e2e-tests.yml) are distinct from backend tests and they do of course run with javascript inside a headless browser. Thought I have to mention that E2E tests are lacking, but that is the overall idea.